src/stream.js

```diff
@@ -3,5 +3,90 @@
 const logger = require('./logger');
 
 module.exports.handler = async (event) => {
+	event = decodeEvent({event});
 	logger.debug({event}, 'Event received');
 };
+
+class Question {
+	static get dbKeys() {
+		return ['date', 'channel'];
+	}
+
+	get answers() {
+		return this._answers;
+	}
+
+	get channel() {
+		return this._channel;
+	}
+
+	get date() {
+		return this._date;
+	}
+
+	constructor(image) {
+		this._date = image.date.S;
+		this._channel = image.channel.S;
+		this._answers = [];
+		Object.keys(image).forEach(key => {
+			if (!Question.dbKeys.includes(key) && image[key] && image[key].M) {
+				this._answers.push({
+					name: key,
+					answer: image[key].M.answer && image[key].M.answer.BOOL,
+					timestamp: image[key].M.timestamp && image[key].M.timestamp.N,
+				});
+			}
+		});
+		this._answers.sort(this._compareAnswers);
+	}
+
+	_compareAnswers(a, b) {
+		if (a.timestamp < b.timestamp) {
+			return 1
+		} else if (a.timestamp > b.timestamp) {
+			return -1
+		}
+
+		if (a.name < b.name ) {
+			return 1
+		} else if (a.name > b.name) {
+			return -1;
+		}
+
+		return 0;
+	}
+}
+
+class DynamoDbRecord {
```

Comment:
This is just a basic class that will get the required information for us choose which events to actually decode.